resources/packs/aws/aws.go

```diff
@@ -52,3 +54,13 @@ func GetRegionFromArn(arnVal string) (string, error) {
 	}
 	return parsedArn.Region, nil
 }
+
+func Is400AccessDeniedError(err error) bool {
+	var respErr *http.ResponseError
+	if errors.As(err, &respErr) {
+		if respErr.HTTPStatusCode() == 400 && strings.Contains(respErr.Error(), "AccessDeniedException") {
```

Comment:
Bit weird it's a 400, i'd expect this to be a 403

Reply:
ya, i think it's because access denied, not authorization denied, that its a 400